docs/sphinx/examples/python/tutorials/readout_error_mitigation.ipynb

```diff
@@ -0,0 +1,845 @@
+   "metadata": {},
+   "source": [
+    "## Inverse confusion matrix from single-qubit noise model\n",
+    "\n",
+    "Here we assume that the readout error of each qubit is independent and with same confusion probabilities. The confusion matrix $A_1$ for a single qubit then can be defined as\n",
+    "\n",
+    "$$\n",
+    "A_1 = \\begin{bmatrix}\n",
+    "P(0|0) & P(0|1)\\\\\n",
+    "P(1|0) & P(1|1)\n",
+    "\\end{bmatrix}\n",
+    "$$\n",
+    "\n",
+    "where $P(y|x)$ is the probability of observing state $|y\\rangle$ when measuring true state $|x\\rangle$. Using $A_1$ the full $2^n \\times 2^n$ confusion matrix $A$ can be written as \n",
+    "\n",
+    "$$\n",
+    "A = A_1 \\otimes \\dots \\otimes A_1\n",
+    "$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "e6ceb0a9-c2ec-4fcd-bb23-457890a8b57e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0 becomes {'0': 924, '1': 100}\n",
+      "1 becomes {'0': 104, '1': 920}\n"
+     ]
+    }
+   ],
+   "source": [
+    "single_qubit_labels = [[0], [1]]\n",
+    "single_qubit_states = [\"0\", \"1\"]  # Used for dictionary keys\n",
+    "\n",
+    "# we use noise_1 which has same bitflip probability for all qubits\n",
+    "results = [\n",
+    "    cudaq.sample(kernel, 1, label, shots_count=shots, noise_model=noise_1)\n",
+    "    for label in single_qubit_labels\n",
+    "]\n",
+    "\n",
+    "for i, state in enumerate(single_qubit_states):\n",
+    "    res = dict(list(results[i].items()))\n",
+    "    print(f\"{state} becomes {res}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "9f030f99-0ad9-4f89-8d99-a570b3c5e745",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[0.90234375 0.1015625 ]\n",
+      " [0.09765625 0.8984375 ]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Construct confusion matrix for single qubit\n",
+    "\n",
+    "A_1 = np.zeros((2, 2))\n",
+    "for i, state in enumerate(single_qubit_states):\n",
+    "    true_state = int(state, 2)\n",
+    "    for key, val in results[i].items():\n",
+    "        observed_state = int(key, 2)\n",
+    "        A_1[observed_state][true_state] = val / shots\n",
+    "\n",
+    "print(A_1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "cbc4a228-935f-413c-9ad3-b6e26f22e802",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[476   3   0   1  12  19   9 501]\n"
+     ]
+    }
+   ],
+   "source": [
+    "A = reduce(np.kron, [A_1] * n_qubits)  # joint confusion matrix\n",
+    "A_pinv = np.linalg.pinv(A)  # Generate psuedoinverse confusion matrix.\n",
+    "mitigated = np.array(np.dot(A_pinv, noisy_res_1), dtype=int)  # Obtain mitgated counts\n",
+    "print(mitigated)\n",
```

Comment:
Could you please add some description text for this `print(mitigated)` and the other below (corrected for negative counts) so that we can understand the output easily?